templates/visuals/default/src/visual.ts

```diff
@@ -61,13 +61,13 @@ export class Visual implements IVisual {
     public update(options: VisualUpdateOptions) {
         this.settings = Visual.parseSettings(options && options.dataViews && options.dataViews[0]);
         console.log('Visual update', options);
-        if (typeof this.textNode !== "undefined") {
+        if (typeof this.textNode !== undefined) {
```

Comment:
Whether it enough to just use the following construction? I thought TS now lets it
if (this.textNode !== undefined)